grakn/rpc/database_manager.py

```diff
@@ -37,11 +39,11 @@ def create(self, name: str) -> None:
         pass
 
     @abstractmethod
-    def delete(self, name: str) -> None:
+    def get(self, name: str) -> Database:
```

Comment:
In Core, the primary purpose of `get` is to retrieve a Database so that you can `delete` it. In Cluster, it is quite a bit more interesting.